controllers/users.js

```diff
@@ -607,6 +607,26 @@ const setInDiscordScript = async (req, res) => {
   }
 };
 
+const removeGitHubTokenFromAllUsers = async (req, res) => {
+  try {
+    const users = await userQuery.usersWithGitHubToken();
+    const length = users.length;
+
+    if (length === 0) {
+      return res.status(500).json({ message: "No users found with github Token!" });
+    }
```

Comment:
What i thought was, after we have already ran this script once, tokens will be removed from all the users. Now, if again i try to run this script, it should give internal server error, because this action is not permitted now.
Because after you run the script once, there will be 0 users with tokens field.

Reply:
> What i thought was, after we have already ran this script once, tokens will be removed from all the users. Now, if again i try to run this script, it should give internal server error, because this action is not permitted now. Because after you run the script once, there will be 0 users with tokens field.

then in that case return 404 with a appropriate message in the response . returning 500 will cause more confusion